sample/src/androidTest/java/com/schibsted/spain/barista/sample/AssertionsTest.java

```diff
@@ -305,4 +311,9 @@ public void checkViewHasNotFocus() throws Exception {
     assertNotFocused(R.string.edittext_with_no_focus);
     assertNotFocused("EditText with no focus");
   }
+
+  @Test
+  public void checkExpectedText() throws Exception {
```

Comment:
A small comment there: can you move this test near the other test methods that check for texts? Just to have them grouped 🙌

Reply:
Of course!